containers/ecr-viewer/guides/setup.md

```diff
@@ -144,6 +144,144 @@ curl --location '<DIBBS_URL>/ecr-viewer/api/migrate-db' \
 --form 'init_admin_email=<email>'
 ```
 
+## Database Documentation
+
+This section provides an overview of the database schema used by the DIBBS eCR Viewer, including both the core and extended schemas, and information about supported database types.
+
+### Supported Database Types
+
+The DIBBS eCR Viewer supports the following relational database types for storing metadata:
+
+- **SQL Server**
+- **PostgreSQL**
+
+### Core Schema
+
+The core schema contains essential data related to eCRs and their associated conditions and rule summaries. This schema is fundamental to the operation of the eCR Viewer.
+
+#### `ecr_data` Table
+
+This table stores the primary eCR data.
+
+| Column Name            | Data Type      | Description                                                                |
+| :--------------------- | :------------- | :------------------------------------------------------------------------- |
+| `eicr_id`              | `varchar(200)` | Primary key, unique identifier for the eICR.                               |
+| `set_id`               | `varchar(255)` | Identifier for a set of related eICRs.                                     |
+| `eicr_version_number`  | `varchar(50)`  | Version number of the eICR.                                                |
+| `fhir_reference_link`  | `varchar(255)` | Reference link to the FHIR resource.                                       |
+| `last_name`            | `varchar(255)` | Last name of the patient (non-nullable).                                   |
+| `first_name`           | `varchar(255)` | First name of the patient (non-nullable).                                  |
```

Comment:
how are these markdown tables generated? any way to make this automagic (or near) off of the database tables or types themselves?